index.html

```diff
@@ -1,86 +1,85 @@
                                                        about="popupopen"
                                                        width="50" height="24"></button>
   </div>
-  <div class="header__iphone-img">
-    <img src="img/iphone-hand-mobile@1x.png"
-         alt="iphone-img" width="258" height="286">
+  <div class="iphone-img">
+    <img src="http://placehold.it/150x100" alt="iphone-img" width="258" height="286">
   </div>
-  <div class="header__present">
-    <a class="download__app" href="#">Скачать приложение</a>
-    <ul class="platform__icon">
-      <li><a href=""><img alt="apple" src="img/logo-apple.svg" width="47" height="56"></a></li>
-      <li><a href=""><img alt="android" src="img/logo-android.svg" width="43" height="47"></a></li>
-      <li><a href=""><img alt="windows" src="img/logo-microsoft.svg" width="44" height="41"></a></li>
+  <div class="presentation">
+    <a class="presentation__download" href="#">Скачать приложение</a>
+    <ul class="presentation__platform-icon">
+      <li><a href=""><img src="http://placehold.it/150x100" alt="apple" width="47" height="56"></a></li>
+      <li><a href=""><img src="http://placehold.it/150x100" alt="android" width="43" height="47"></a></li>
+      <li><a href=""><img src="http://placehold.it/150x100" alt="windows" width="44" height="41"></a></li>
     </ul>
     <p>Доступно для iPhone, iPad, Android,<br>Windows Phone, OS X, Windows 8</p>
   </div>
 </header>
-<main class="page__main">
+<main class="main">
   <div class="main__title">
     <h2>Устали от серости<br>мегаполиса?</h2>
     <h2>Нам есть, что вам<br>предложить!</h2>
   </div>
-  <div class="main__about">
-    <div class="main__about--text text--first">
+  <div class="about">
+    <div class="about__text">
       <h3>Поднимает настроение</h3>
       <p>
         Приложение позволит вам победить
         осеннюю хандру и депрессию буквально в
         несколько кликов!
       </p>
     </div>
-    <div class="main__about--text text--second">
+    <div class="about__text">
       <h3>Меняет мир вокруг</h3>
       <p>
         Сделайте снимок и украсьте его смайликом
         или текстовой подписью, чтобы усилить
         эффект
       </p>
     </div>
-    <div class="main__about--text text--third">
+    <div class="about__text">
       <h3>Заводит новых друзей</h3>
       <p>Ставьте лайки, комментируйте,
         делитесь фотографиями с друзьями и
         заводите новых</p>
     </div>
   </div>
-  <div class="main__reviews">
-    <div class="reviews__text">
-      <h4>Николай Петров</h4>
+  <div class="reviews">
+    <ul class="reviews__text">
+      <li><h4>Николай Петров</h4>
       <p>«Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я
-        познакомился со своей будущей женой в комментариях к выложенной фотографии!»</p>
-    </div>
+        познакомился со своей будущей женой в комментариях к выложенной фотографии!»</p></li>
+    </ul>
     <div class="reviews__btn">
```

Comment:
Это не кнопка, а блок с кнопками. Хорошее название `reviews__button-list` или `reviews__buttons`.
```suggestion
    <ul class="reviews__buttons">
```